docs/section6/Rule6-5.md

```diff
@@ -0,0 +1,30 @@
+
+# Lighting - Rule 6-5
+
+**Rule ID:** 6-5  
+**Rule Description:** Where a lighting system has been designed and submitted with design documents, lighting power shall be determined in accordance with Sections 9.1.3 and 9.1.4.  
+**Appendix G Section:** Section G3.1-6(b) Modeling Requirements for the Proposed Design  
+**Appendix G Section Reference:** None  
+
+**Applicability:** All required data elements exist for P_RMR  
+**Applicability Checks:** None  
+**Manual Check:** None  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** None  
+## Rule Logic: 
+
+- For each building_segment in the proposed model: ```building_segment_p in P_RMR.building.building_segments:```  
+
+  - For each thermal_block in building segment: ```thermal_block_p in building_segment_p.thermal_blocks:```  
+
+    - For each zone in thermal block: ```zone_p in thermal_block_p.zones:```  
+
+      - For each space in zone: ```space_p in zone_p.spaces:```  
+
+        - Get interior lighting in U_RMR: ```interior_lighting_b = match_data_element(U_RMR, InteriorLightings, space_p.interior_lighting.id)```
+
+          **Rule Assertion:** 
+
+          - Case 1: Lighting power in the proposed RMR is as designed and cannot be verified by RCT: ```if space_p.interior_lighting == interior_lighting_b: CAUTION```
+
```

Comment:
The RDS looks good to me except I am struggling with "CAUTION" outcome. RCT will be checking numerous aspects of P-RMR to confirm that they are equal to the corresponding aspects of U-RMR, so will generate numerous "CAUTIONS" for each project. For example, on a project with 100 spaces that all have the same LPD in the U-RMR and P-RMR (the best case outcome), RCT will generate 100 "CAUTION" flags. I think this will result in these flags being ignored by human reviewers working for AHJs / RAs. I recommend making it "PASS", and providing an umbrella statement in the RCT report to verify that LPD modeled in the proposed design reflects design documents.

Reply:
@mkarpman the RCT will generate two reports, one high level report which summarizes all rules that were pass/fail/caution and a second more detailed report where details for each space would be shown. At the high level report, this will show up as just one fail/caution. 
If we decide that 6-5 should be caution, then I think that 6-4 should be caution as well.

Reply:
@supriyagoel we have RCT project checking report described in Section 7.4.3. There is not much in there currently, perhaps we should propose updates for this section and vet it with the committee? The section talks about RCT reporting schema. Do we need to be developing it also?